webpack/components/extensions/HostDetails/Cards/__tests__/errataOverviewCard.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render } from 'react-testing-lib-wrapper';
+import ErrataOverviewCard from '../ErrataOverviewCard';
+
+test('does not show piechart when content facet is not set', () => {
+  const { getByLabelText } = render(<ErrataOverviewCard />);
+  expect(getByLabelText('errataChart').not.toBeInTheDocument());
```

Comment:
```suggestion
  expect(getByLabelText('errataChart')).not.toBeInTheDocument();
```
you just need to move the closing parentheses :)

Also, if you use `.not` then I prefer `queryByLabelText` over `getByLabelText`.  `get` will error if it can't find the element, whereas `query` will just return `undefined`.  So if you use `get` here, the `not` will never be reached.

Reply:
Thanks for the pointer, it got me started in the right direction and I learned something about the testing functions :)